plugins/modules/zos_encode.py

```diff
@@ -352,6 +359,7 @@ def run_module():
         backup=dict(arg_type="bool", default=False, required=False),
         backup_name=dict(arg_type="data_set_or_path", required=False, default=None),
         backup_compress=dict(arg_type="bool", required=False, default=False),
+        tmphlq=dict(type='qualifier_or_empty', required=False, default=None),
```

Comment:
There is confusion... what I thought I saw was 'tmphlq' is a string/parameter, and tmp_hlq is a dictionary. As I look more closely, I'm seeing them intermixed. I'll try to put comment notes where I see tmphlq as a dictionary.

Reply:
Resolving because I created issue https://github.com/ansible-collections/ibm_zos_core/issues/387 - it was a good comment and point, please review the issue as well @fernandofloresg , also see the other work item that came from this PRs review https://github.com/ansible-collections/ibm_zos_core/issues/386